### PR TITLE
Added support for other than public GitHub URL

### PR DESCRIPTION
**Pull request by Hi-Fi**
Refactoring a bit

### CODE REVIEW

File: runner/entrypoint.sh [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+if [ -z "${GITHUB_URL}" ]; then
+  echo "Working with public GitHub" 1>&2
+  GITHUB_URL="https://github.com"
```
Comment (mumoshu) [inline runner/entrypoint.sh:5]: I think we need a trailing `/` here

File: runner/entrypoint.sh [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+else
+  echo "Github endpoint URL ${GITHUB_URL}"
```
Comment (mumoshu) [inline runner/entrypoint.sh:7]: Would you mind adding additional validation in the controller that ensures GITHUB_URL envvar to have the trailing slash?

File: runner/entrypoint.sh (hunk 1 above)
Comment (Hi-Fi) [reply runner/entrypoint.sh:5]: True, to make runner work with older manager

File: runner/entrypoint.sh (hunk 2 above)
Comment (Hi-Fi) [reply runner/entrypoint.sh:7]: There needs to be some more logic added too, as client's baseURL is by default api.github.com, not github.com.

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks a lot for your continued support 👍

**Pull request merged**